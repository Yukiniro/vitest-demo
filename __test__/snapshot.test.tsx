import React from "react";
import { render, screen } from "@testing-library/react";
import { test, expect } from "vitest";
import matchers from "@testing-library/jest-dom/matchers";
import Button from "../src/button";

globalThis.IS_REACT_ACT_ENVIRONMENT = true;

expect.extend(matchers);

test("Button snapshot", () => {
  render(<Button text="button" />);
  expect(screen.getByRole("button")).toMatchSnapshot();
});
