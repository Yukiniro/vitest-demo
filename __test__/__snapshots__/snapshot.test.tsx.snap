// Vitest Snapshot v1

exports[`button-snapshot 1`] = `
<button>
  button
</button>
`;
